lightning/src/ln/outbound_payment.rs

```diff
@@ -665,32 +667,31 @@ impl OutboundPayments {
 		}
 
 		macro_rules! abandon_with_entry {
-			($payment_id: expr, $payment_hash: expr, $payment: expr, $pending_events: expr) => {
+			($payment_id: expr, $payment: expr, $pending_events: expr) => {
 				if $payment.get_mut().mark_abandoned().is_ok() && $payment.get().remaining_parts() == 0 {
 					$pending_events.lock().unwrap().push(events::Event::PaymentFailed {
 						payment_id: $payment_id,
-						payment_hash: $payment_hash,
+						payment_hash,
 					});
 					$payment.remove();
 				}
 			}
 		}
```

Comment:
I guess we similarly don't need to pass `payment_id` and `pending_events` to the macro?